tensorrt_llm/_torch/modules/mla.py

```
@@ -0,0 +1,2071 @@
+
+
+@mla_dsa_proj.register_fake
+def _mla_dsa_proj_fake(
+    hidden_states: torch.Tensor,
+    position_ids: Optional[torch.Tensor],
+    layer_idx: str,
+) -> List[torch.Tensor]:
+    # Under torch compile _should_use_short_mha is False, so the result is
+    # always 9 tensors (4 attention inputs + 5 indexer intermediates, with
+    # q_scale as the 9th carried for the FP4 dispatch).
+    metadata, mla_layer = _extract_mla_extra_attrs(layer_idx)
```

Ruff (0.15.18)

[warning] Unpacked variable `metadata` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)